index.html

```diff
@@ -322,21 +322,12 @@
           for <a>pointerlockerror</a> events.</p>
         </dd>
 
-        <dt>readonly attribute Element? pointerLockElement</dt>
-        <dd>
-          <dfn title="pointerLockElement"></dfn>
-
-          <p>Returns the element set as the <a>target</a> for mouse events
-          while the pointer is locked. Null if lock is pending, pointer is
-          unlocked, or if the target is in another document.</p>
-        </dd>
-
         <dt>void exitPointerLock ()</dt>
         <dd>
           <dfn title="exitPointerLock"></dfn>
 
           <p>Initiates an exit from pointer lock state if currently locked
-          to a target in this document,
+          to a target in this shadow-including document,
```

Comment:
Hmm. The same problem is here.
DOM Standard defines "in *a* shadow-including document", but we do not have a definition of "a shadow including document", so we can not use "*this* shadow-including document" here.